### DISCUSSION TITLE
How to use eslint < 9.1

### DISCUSSION THREAD

**Discussion by root9464** [eslint/eslint#18411] [Q&A]
When I run the command `nx client --init`, the config ".mjs" is created for me. how can I use the old config and initialize the old eslint?

**Comment by Rec0iL99** [comment]
Hi @root9464, we don't maintain the `nx` tool and I'm guessing the tool uses the latest version of `create-config`. If you want the `create-config` utility to generate an `eslintrc` config (old config style), you would have to use an older version of the utility.

```bash
npm init @eslint/config@0.4.6
```

**Reply by root9464** [reply]
sorry I made a mistake in `npx`
yes it solves my problem quite well thank you
